### PR TITLE
feature: add dark theme for layout and nft gallery

### PR DESCRIPTION
**Pull request by SorinC6**
task: https://emurgo.atlassian.net/browse/YOEXT-1223

Added fixes from: https://emurgo.atlassian.net/browse/YOEXT-1079

### CODE REVIEW

**Comment by emurgo-rnd** [comment]
[YOEXT-1223: Add dark theme support for main Layout and Nft Galley ](https://emurgo.atlassian.net/browse/YOEXT-1223)

**Review by SorinC6 — COMMENTED** [review]
/check

File: packages/e2e-tests/package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "@emurgo/cardano-serialization-lib-nodejs": "^12.0.0-alpha.26",
     "bignumber.js": "^9.1.2",
     "chai": "^4.3.10",
-    "chromedriver": "^126.0.4",
+   "chromedriver": "^126.0.4",
```
Comment (zuzunker) [inline packages/e2e-tests/package.json]:
```suggestion
    "chromedriver": "^126.0.4",
```

**Review by Nebyt — DISMISSED** [review]

**Review by SorinC6 — COMMENTED** [review]
/check

**Review by Nebyt — APPROVED** [review]

**Review by zuzunker — APPROVED** [review]

**Pull request merged**